deploy/helm/flightctl/templates/flightctl-db-admin-secret.yaml

```
@@ -0,0 +1,28 @@
+{{- $namespaces := list .Release.Namespace }}
+{{- if .Values.global.internalNamespace }}
+{{- $namespaces = append $namespaces .Values.global.internalNamespace }}
+{{- end }}
+{{- $existingSecret := (lookup "v1" "Secret" .Release.Namespace "flightctl-db-admin-secret") }}
+{{- $masterPassword := "" }}
+{{- if $existingSecret }}
+  {{- $masterPassword = (index $existingSecret.data "masterPassword") }}
+{{- else }}
+  {{- $masterPasswordRaw := (randAlphaNum 20) }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)